packages/perseus-editor/src/index.ts

```diff
@@ -14,11 +14,12 @@ import "./styles/perseus-editor.less";
 // eslint-disable-next-line import/order
 import {Widgets, widgets} from "@khanacademy/perseus";
 import AllEditors from "./all-editors";
+import ContentPreview from "./content-preview";
```

Comment:
Not a required change, but we can export this component in a single line:

```ts
export {default as ContentPreview} from "./content-preview";
```

Reply:
Whoops! Used the wrong export section. I can see now that section below is there because some work is done to `AllEditors` and `widgets` before exporting. Moved the ContentPreview export to the top with the other exports. Thanks!